lib/less_builder.dart

```diff
@@ -56,14 +58,15 @@ class LessBuilder implements Builder {
 
     final String package = inputId.package;
     final String extension = inputId.extension;
+    final String inputPath = await _filePathFor(inputId);
 
     if (extension.toLowerCase() == '.less') {
       final DotLessBuilder transformer = DotLessBuilder()
-        ..createFlags(options, path_lib.dirname(inputId.path))
+        ..createFlags(options, path_lib.dirname(inputPath))
         ..inputContent = await buildStep.readAsString(inputId);
 
       final AssetId outputId = inputId.changeExtension(DOT_LESS_TO);
-      log.info(
+      log.fine(
```

Comment:
There are a lot of build steps in big angular project. None of them add info-logs on EVERY step